controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -105,6 +105,7 @@
             .put(WireCommands.ReadTable.class, ImmutableSet.of(WireCommands.TableRead.class))
             .put(WireCommands.ReadTableKeys.class, ImmutableSet.of(WireCommands.TableKeysRead.class))
             .put(WireCommands.ReadTableEntries.class, ImmutableSet.of(WireCommands.TableEntriesRead.class))
+            .put(WireCommands.FlushToStorage.class, ImmutableSet.of(WireCommands.StorageFlushed.class))
```

Comment:
This needs to go in `AdminSegmentHelper`.

Reply:
I've changed this.

Reply:
It's still here. I'm referring to this entry in this map.

Reply:
I've removed the entry now.